# Generated by Django 3.1.4 on 2021-01-13 18:06

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('e_kolej_management_app', '0001_initial'),
    ]

    operations = [
        migrations.CreateModel(
            name='ApplyException',
            fields=[
                ('id', models.AutoField(primary_key=True, serialize=False)),
                ('gender', models.CharField(max_length=255)),
                ('address', models.TextField()),
                ('level', models.CharField(max_length=255)),
                ('scholarship', models.CharField(max_length=255)),
                ('family_income', models.CharField(max_length=255)),
                ('family_dependent', models.CharField(max_length=255)),
                ('family_problem', models.CharField(max_length=255)),
                ('health_problem', models.CharField(max_length=255)),
                ('apply_status', models.IntegerField(default=0)),
                ('created_at', models.DateTimeField(auto_now_add=True)),
                ('updated_at', models.DateTimeField(auto_now_add=True)),
                ('admin', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
                ('student_id', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='e_kolej_management_app.students')),
            ],
        ),
    ]
